src/utils/accounts.js

```diff
@@ -96,16 +100,14 @@ export const getAccountName = (accountType: AccountTypes, accounts: Accounts): s
   }
 };
 
-export const findAccountByAddress = (
-  address: string,
-  accounts: Accounts,
-): ?Account => {
+export const findAccountByAddress = (address: string, accounts: Accounts): ?Account => {
   return accounts.find(account => addressesEqual(address, getAccountAddress(account)));
 };
 
-export const findAccountById = (
-  accountId: string,
-  accounts: Accounts,
-): ?Account => {
+export const findAccountById = (accountId: string, accounts: Accounts): ?Account => {
   return accounts.find(({ id }) => id === accountId);
 };
+
+export const getAccountWalletId = (account: Account): string => {
+  return get(account, 'walletId', '');
```

Comment:
Wouldn't it be the same as `account.walletId || ''`? This function requires an `account` argument, and the only caller already does a null check.